MaxText/tests/quantizations_test.py

```diff
@@ -101,7 +101,6 @@ def test_configure_quantization_is_int8(self):
       quant = _configure_quantization(quant_str="int8", mode_str=quant_mode)
       self.assertNotEqual(quant, None)
 
-  @pytest.mark.skip(reason="b/400476456 Tests are currently flaking / failing due to JAX 0.5.1 upgrade")
   def test_aqt_quantization(self):
```

Comment:
LGTM pending fix of the test: https://github.com/AI-Hypercomputer/maxtext/actions/runs/14480038865/job/40616108017?pr=1581